test/distributed/test_functional_api.py

```diff
@@ -232,6 +232,22 @@ def setUp(self):
         super().setUp()
         self._spawn_threads()
 
+    @parametrize("device", ["cpu", "cuda"])
+    def test_broadcast(self, device):
+        if device == "cuda":
+            if torch.cuda.device_count() < self.world_size:
```

Comment:
any reason not to use     @skip_if_lt_x_gpu
 here?

Reply:
I was just following the pattern in the other tests, but I would guess the reason here is we don't want to skip when device="cpu"?